cmd/eskip/media_test.go

```diff
@@ -64,16 +64,6 @@ func TestValidateSelectMedia(t *testing.T) {
 		in      *medium
 		out     *medium
 	}{{
-
```

Comment:
as this case will never happen... this function takes care of this :)

    src/github.com/zalando/skipper/cmd/eskip/eskip.go:74